eviatar/main.tf

```diff
@@ -0,0 +1,47 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+resource "aws_instance" "windows_instance" {
+  ami           = "ami-07d9456e59793a7d5"  # Replace with the Windows AMI ID of your choice

```

Comment:
I want to use this dynamically... 
Imaggine swapping out the human operator for a Configuration Management system that stores values for various environments

Reply:
To make the AMI ID dynamic, I could use variables or pull the AMI ID from a configuration management system or parameter store like AWS SSM Parameter Store. This would allow the script to dynamically select the correct AMI based on the environment.